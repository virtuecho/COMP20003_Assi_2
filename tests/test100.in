Windsor
Ascot Vale
Parkville
Kensington
Melbourne
Richmond
Ascot
Red Hill
Brunswick
Kensington Gardens
Kensington Park
Parkvile
