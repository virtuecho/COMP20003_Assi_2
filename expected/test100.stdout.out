Windsor --> 4 records found
Ascot Vale --> 1 records found
Parkville --> 2 records found
Kensington --> 5 records found
Melbourne --> 1 records found
Richmond --> 6 records found
Ascot --> 5 records found
Red Hill --> 8 records found
Brunswick --> NOT FOUND
Kensington Gardens --> 1 records found
Kensington Park --> 1 records found
Parkvile --> NOT FOUND
